src/client/components/MealList/MealList.jsx

```diff
@@ -1,26 +1,36 @@
-import React, { useContext } from "react";
-import Meal from "../Meal/Meal";
-import { MealContext } from "../Context/MealContext";
+import React from "react";
+import { Link } from "react-router-dom";
+import { useMealContext } from "../Context/MealContext";
+import MealSearch from "../MealSearch/MealSearch";
+import MealSort from "../MealSort/MealSort";
+import { useSortedMealContext } from "../Context/MealSortContext";
 import "../Meal/meal.css";
 const MealList = () => {
-  const usingContext = useContext(MealContext);
+  const { currentMeals } = useMealContext();
+  const { sortedtCurrentMeals } = useSortedMealContext();
```

Comment:
Should these 2 live together in the same context? 
Is meals or sorted meals really 2 completely different contexts? Or is it the same exact meals, just sorted or not sorted?

Reply:
These 2 different context I made,as sorting mutate the original array so I thought to make 2 different context.